PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,10 @@
+<!-- 
+The PR title should match: <type>(optional <scope>): <description>.
+
+Please also ensure all commits in this PR comply with our conventional commits specification:
+https://docs.google.com/document/d/1SYUo9G7qZ_jdoVXpUVamS5VCgHmtZ0QA-wZxKoMS-C0 
```

Comment:
People outside of Canonical can't view this so it's useless for an open source project. Point this to the main website.

Reply:
This doc is actually fully public, the tag on it says its open to everyone, canonical or not

Reply:
Ah ok, go for it.

Reply:
This could actually be transplanted into the dev docs? https://juju.is/docs/dev

Reply:
@tmihoc :point_up: